make_digest.py

```diff
@@ -1,12 +1,21 @@
 from bs4 import BeautifulSoup
 import requests
+from datetime import date
 
 # Where's the list of articles to include in each section?
 DIGEST_IN = "digest-in.txt"
 
 # Where should the code be output?
 DIGEST_OUT = "digest-out.txt"
 
+# What are the URL referral tracking parameters for this digest?
+TODAY = date.today()
+UTM_CAMPAIGN = "utm_campaign=" + "digest"
+UTM_MEDIUM = "&utm_medium=" + "email"
+UTM_SOURCE = "&utm_source=" + "mailchimp"
+UTM_CONTENT = "&utm_content=" + today.strftime("%b-%d-%Y")
```

Comment:
This is a style nit but in my experience these all-caps variables are declared for constants that won't change on different executions of the script, unlike the date. It's up to you and has extremely little impact but another option here might be to declare constants like

```
UTM_CAMPAIGN = "digest"
UTM_MEDIUM = "email"
UTM_SOURCE = "mailchimp"
```

and then format them into a string with `?` and `&`s in `format_link`. (`urllib.parse.urlencode` would do that all for you, but if you really wanted to avoid the import, you could also consider something like `"&".join({ "utm_campaign" : UTM_CAMPAIGN, "utc_medium": UTM_SOURCE, ... })`